charts/cloudzero-agent/templates/validatorcm.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    {{- include "cloudzero-agent.server.labels" . | nindent 4 }}
+  name: {{ include "cloudzero-agent.validatorConfigMapName" . }}
+  namespace: {{ include "cloudzero-agent.namespace" . }}
+  {{- with .Values.prometheusConfig.configMapAnnotations }}
+  annotations:
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+data:
+  validator.yml: |-
+    versions:
+      chart_version: {{ .Chart.Version }}
+      agent_version: {{ .Values.server.image.tag }}
+
+    logging:
+      level: info
+      location: ./cloudzero-agent-validator.log
+
+    deployment:
+      account_id: {{ .Values.cloudAccountId }}
+      cluster_name: {{ .Values.clusterName }}
+      region: {{ .Values.region }}
+
+    cloudzero:
+      host:  https://{{ .Values.host }}
+      credentials_file: /etc/config/prometheus/secrets/value
+      disable_telemetry: false
+
+    prometheus:
+      {{- if .Values.validator.serviceEndpoints.kubeStateMetrics }}
+      kube_state_metrics_service_endpoint: http://{{ .Values.validator.serviceEndpoints.kubeStateMetrics }}/
+      {{- else }}
+      kube_state_metrics_service_endpoint: http://{{- if .Release.Name }}{{.Release.Name}}-{{- end }}kube-state-metrics:8080/
```

Comment:
I think if we really wanted to be safe, we could try to get this value from the subchart directly instead of semi-hardcoding the `kube-state-metrics:8080` bit (see [here, maybe](https://helm.sh/docs/topics/charts/#importing-child-values-via-dependencies)). but that is probably overkill, and I think we may not even use ksm/pne as subcharts in the future anyway. just thought I'd mention it

Reply:
Would this also be true if they didn't deploy the sub-chart? eg. 

**pseudo code**
```
if subchart.enabled
read value
else 
use the value from values.yaml
```

Reply:
good point, yeah that pseudocode seems right to me

Reply:
I think this would require [kube-state-metrics Chart.yaml](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-state-metrics/Chart.yaml) and [prometheus-node-exporter](https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-node-exporter/Chart.yaml) to contain the `export` keyword. It doesn't look that those are available.

Reply:
Thanks for the guidance on this @dmepham. As discussed, we can resolve this comment for now, and address in an upcoming PR